packages/components/src/globals/scss/vendor/@carbon/elements/scss/type/_inlined/_styles.scss

```diff
@@ -468,6 +468,44 @@ $display-04: (
   ),
 ) !default;
 
+/// Used to compare with `$tokens` map so tokens can be reset to their default
```

Comment:
@mattrosno out of curiosity, why is this file duplicated 4 times in this PR?

> packages/components/src/globals/scss/vendor/@carbon/type/scss/_styles.scss 
> packages/components/src/globals/scss/vendor/@carbon/type/scss/_inlined/_styles.scss
> packages/components/src/globals/scss/vendor/@carbon/elements/scss/type/_styles.scss
> packages/components/src/globals/scss/vendor/@carbon/elements/scss/type/_inlined/_styles.scss

Reply:
We inline elements packages for teams who can't configure their Sass loader to import directly from `node_modules`. Those duplicated files are generated in our build. In this example, I'm making edits to the `_styles.scss` from the `@carbon/type` package and the build does the rest.